src/views/box_view.rs

```diff
@@ -212,12 +213,25 @@ impl<T: View> BoxView<T> {
 impl<T: View> ViewWrapper for BoxView<T> {
     wrap_impl!(self.view: T);
 
+    fn wrap_draw(&self, printer: &Printer) {
+        let mut printer = printer.clone();
+        // Set values hard here, since cropping would allow smaller size if fixed or at least set
+        printer.size.x = self.size.x.result((printer.size.x, printer.size.x));
+        printer.size.y = self.size.y.result((printer.size.y, printer.size.y));
+        self.view.draw(&printer);
+        self.view.draw(&printer);
```

Comment:
Duplicated line?

Reply:
Oh yeah, thanks for the hint, overlooked this while rebasing